code/datums/diseases/transformation.dm

```diff
@@ -388,3 +388,52 @@
 	stage5	= list("<span class='userdanger'>You have become one of Legion. You are one with the Necropolis now, and have no other loyalties. Serve well.</span>")
 	new_form = /mob/living/simple_animal/hostile/asteroid/hivelord/legion/tendril
 	infectable_biotypes = list(MOB_ORGANIC, MOB_INORGANIC, MOB_UNDEAD)
+
+/datum/disease/transformation/grod
+	name = "Grod Mutation Metamorphisis"
+	agent = "Crown Microbes"
+	desc = "Who knows why Crowns do this?"
+	danger = DISEASE_MINOR
+	visibility_flags = 0
+	stage_prob = 35
+	cures = list(/datum/reagent/consumable/sodiumchloride)
+	cure_chance = 10
+	stage1	= list("Your joints feel soft.")
+	stage2	= list("Your skin feels soft.")
+	stage3	= list("You feel slugish.")
+	stage4	= list("You feel sleepy.")
+	stage5	= list("You want to sleep...")
+	new_form = /mob/living/carbon/human/species/grod
+	infectable_biotypes = list(MOB_ORGANIC, MOB_INORGANIC, MOB_UNDEAD)
+
+/datum/disease/transformation/grod/do_disease_transformation(mob/living/affected_mob) //Pretty much a duplicate of regular expression, inserts caccoon code
+	if(istype(affected_mob, /mob/living/carbon) && affected_mob.stat != DEAD)
+		if(stage5)
+			to_chat(affected_mob, pick(stage5))
+		if(QDELETED(affected_mob))
+			return
+		if(affected_mob.notransform)
+			return
+		affected_mob.notransform = 1
+		for(var/obj/item/W in affected_mob.get_equipped_items(TRUE))
+			affected_mob.dropItemToGround(W)
+		for(var/obj/item/I in affected_mob.held_items)
+			affected_mob.dropItemToGround(I)
+		var/mob/living/carbon/human/species/grod/new_mob = new new_form(affected_mob.loc) //type cast, from normal proc, to access dna features
+		if(istype(new_mob))
```

Comment:
why is this needed?

Reply:
this is copied from the parent proc, i just didn't call so I could inject some changes.
I guess I'll change it, however.